src/metpy/calc/thermo.py

```diff
@@ -1023,16 +1023,36 @@ def saturation_vapor_pressure(temperature):
     .. math:: 6.112 e^\frac{17.67T}{T + 243.5}
 
     """
-    # Converted from original in terms of C to use kelvin.
-    return mpconsts.nounit.sat_pressure_0c * np.exp(
-        17.67 * (temperature - 273.15) / (temperature - 29.65)
-    )
+
+    def liquid(temperature):
+        # Converted from original in terms of C to use kelvin.
+        return mpconsts.nounit.sat_pressure_0c * np.exp(
+            17.67 * (temperature - 273.15) / (temperature - 29.65)
+        )
+
+    def ice(temperature):
+        # Alduchov and Eskridge (1996)
+        return mpconsts.nounit.sat_pressure_0c * np.exp(
+            22.587 * (temperature - 273.16) / (temperature + 0.7)
+        )
+
+    if phase == 'liquid':
+        return liquid(temperature)
+    elif phase == 'ice':
+        return ice(temperature)
+    else:
+        assert phase == 'temperature-dependent'
+        alpha = np.zeros_like(temperature, dtype=float)
+        t_sel = (temperature > 250.16) & (temperature < 273.16)
+        alpha[t_sel] = ((temperature[t_sel] - 250.16) / (273.16 - 250.16)) ** 2
+        alpha[temperature >= 273.16] = 1
```

Comment:
Would it be worth adding these temperatures to `metpy.constants` and using those here?